lib/CurrencyExchangeRateFields/CurrentExchangeRate/CurrentExchangeRate.js

```diff
@@ -60,11 +60,13 @@ const CurrentExchangeRate = ({
   }
 
   return (
-    <KeyValue
-      data-testid="current-exchange-rate"
-      label={label}
-      value={exchangeRate?.exchangeRate}
-    />
+    <div style={{ width: '100%' }}>
```

Comment:
plz no inline styles. Use CSS or just wrap it in the `<Layout>` component with appropriate class ("full") - https://ux.folio.org/storybook/?path=/story/layout--available-classes